Create ui-c8y-1023-88-2-agent-chat-show-error-if-output-tokens-limit-is-reached.md (#4817)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-88-2-agent-chat-show-error-if-output-tokens-limit-is-reached.md

---
date: ""
title: Agent chat shows error when output token limit is reached
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65514
version: 1023.88.2
---

> For any agent you can set maximum output tokens to limit number of tokens generated by model in single response. If you reach single response max tokens limit, error is displayed. You can continue this conversation, as max tokens limit applies to single response, not summary of output tokens through the conversation.

Reviewer: ```suggestion
For any AI agent, you can set a maximum output token limit to control the number of tokens the model generates in a single response and affect chat process with it. Previously, if this limit was reached, the response would be incomplete without clear feedback. Now, when the output token limit is reached for a single response, an error message is displayed to inform you of the limitation. You can continue the conversation after this error, as the token limit applies only to individual responses, not to the total tokens used across the entire conversation.
```

> For any AI agent, you can set a maximum output token limit to control the number of tokens the model generates in a single response and affect chat process with it. Previously, if this limit was reached, the response would be incomplete without clear feedback. Now, when the output token limit is reached for a single response, an error message is displayed to inform you of the limitation. You can continue the conversation after this error, as the token limit applies only to individual responses, not to the total tokens used across the entire conversation.

Reviewer 2: ```suggestion
For any AI agent, you can set a maximum output token limit to control the number of tokens the model generates in a single response, which has an effect on the chat process. Previously, if this limit was reached, the response would be incomplete without clear feedback. Now, when the output token limit is reached for a single response, an error message is displayed to inform you of the limitation. You can continue the conversation after this error, as the token limit applies only to individual responses, not to the total tokens used across the entire conversation.
```

Reviewer 2: Not sure if this is what you mean @jdre-c8y ?

Reviewer: @BeateRixen  yes, this is it
